run/src/css/Menu.scss

```diff
@@ -0,0 +1,69 @@
+@import "colors";
+@import "variables.scss";
+.menu {
+  margin-right: 3.8%;
+  float: left;
+  min-height: 1px;
+  &__items {
+    display: none;
+  }
+  &__figure {
+    display: block;
+    position: relative;
+    margin-left: 80%;
+    &--first {
```

Comment:
you can accomplish the same result with flex and without using &--first, &--second and &--third